Containerfile

```diff
@@ -32,6 +32,28 @@ RUN --mount=type=cache,dst=/var/cache/libdnf5 \
     --mount=type=secret,id=GITHUB_TOKEN \
     /ctx/build_files/shared/build.sh
 
+# Install Homebrew
+RUN --mount=type=cache,dst=/var/cache/homebrew,uid=1000,gid=1000 \
+    set -eoux pipefail && \
+    useradd -u 1000 -m -s /bin/bash -c "Homebrew Build User" linuxbrew && \
+    mkdir -p /var/home/linuxbrew/.linuxbrew && \
+    chown -R 1000:1000 /var/home/linuxbrew && \
+    mkdir -p /var/cache/homebrew /var/lib/homebrew && \
+    chown -R 1000:1000 /var/cache/homebrew /var/lib/homebrew && \
+    su - linuxbrew -c "bash -c ' \
+        export NONINTERACTIVE=1 && \
+        export HOMEBREW_BREW_GIT_REMOTE=https://github.com/Homebrew/brew && \
+        export HOMEBREW_CORE_GIT_REMOTE=https://github.com/Homebrew/homebrew-core && \
+        export HOMEBREW_NO_AUTO_UPDATE=1 && \
+        curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh | bash \
```

Comment:
We might run into usage limits with pulling from GitHub. Could we incorporate the ghcurl script into this somehow?